cmd/yggctl/main.go

```diff
@@ -131,6 +131,57 @@ func main() {
 			},
 			Action: dispatchAction,
 		},
+		{
+			Name:        "message-journal",
+			Usage:       "Display a list of all worker messages and emitted events",
+			UsageText:   "yggctl message-journal",
+			Description: "The message-journal command sends a dbus call to yggdrasil to retrieve a message journal containing all worker messages and emitted events.",
```

Comment:
D-Bus is an irrelevant implementation detail from the user's perspective. Since
these strings are meant to be user-facing, this description should tell the user
what the operation will/should do, not necessarily how it accomplishes that
operation.

```suggestion
			Description: "The message-journal command retrieves a message list containing all worker messages and emitted events.",
```